### PR TITLE
ready for review 

### PR DESCRIPTION
**Pull request by ecpierce**

### CODE REVIEW

File: data/log2fcspatial.csv [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+Aspergillus flavus,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
+Paracoccidioides lutzii,0,0,-0.286881148,0,0,0,0,0,0,0,-0.101842141,0,0,0,0,0,0,0,0,0,0,0,3.058893689,0,0,0,0,0,0,0,0,0,0,-2.121015401,0,0,0,0,0,2.795859283,0,0,0,0,0,1,0,2.035046947,0,0,0,0,0
+Fusarium solani,0,0.596550475,1.302819272,0,0,0,0,3.380821784,0,0,1.507993024,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,-0.565597176,-1.464417342,0,0,0,0,0,3.795859283,0,0,0,0,0,3.064917477,0,0,4.10780329,0,0,0,0
+Graphium fructicola,0,-0.548601025,0,0,0,0,0,0,0,0,0,0,0,2.983931631,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,-1.777928142,0,0,0,0,0,0,0,0,4.124482031,0,0,0
+Nectria cinnabarina,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,2.058893689,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
+Neonectria neomacrospora,0,0.522840789,0,3.795859283,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,1.746615764,2.293358943,0,0,0,0,0,0,-0.711935357,0,-0.607862903,0,0,0,0,0,-2.20372321,0,2.795859283,0,0,0,0,0,0,0,0,4.795859283,0
+Scopulariopsis brevicaulis,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
```
Comment (taylorreiter) [inline data/log2fcspatial.csv]: missing new line at the end of the file, which is part of a file definition. Added below!
```suggestion
Scopulariopsis brevicaulis,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0

```

File: README.md [hunk 2]
```diff
@@ -1 +1,6 @@
-# hiprfish-imaging
\ No newline at end of file
+# HiPR-FISH Spatial Mapping of Cheese Rind Microbial Communities 
+# :cheese: :microscope: :world_map:
+
+[HiPR-FISH spatial imaging](https://doi.org/10.1038/s41586-020-2983-4) was used to look at the distribution of microbes within five distinct microbial communities growing on the surface of aged washed rind cheeses (link to pub doi). HiPR-FISH was performed by [Kanvas Biosciences](https://www.kanvasbio.com). Imaging data and spatial correlation data is available at (zenodo link).
```
Comment (taylorreiter) [inline README.md]: do you have the DOIs yet for the pub and for zenodo? if so we can add them now, and if not it can be a last minute change!

**Review by taylorreiter — CHANGES_REQUESTED** [review]
This is looking really good! I made on suggestion that will fix the formatting of the CSV. if you add it, make sure you use `git pull` to get those changes synced locally

The notebook was hard to comment on inline so adding observations/suggestions/requests here:
1. Labels will be edited in a figure editor right? if so, the size of text, italics, and periods in the names are fine 
2. will you also make a legend for the side bar colors?
3. can you check whether the red/blue palette is color blind friendly?
4. can you put the sessionInfo block at the bottom of the notebook and then restart the kernel and re-run the notebook and push your changes? it looks like you have extra libraries loaded up that you didn't end up using. 

everything else is great! such a pretty figure!

File: README.md (hunk 2 above)
Comment (ecpierce) [reply README.md]: not yet! but I will add a row in Monday to make sure I do that before I create a release

**Comment by ecpierce** [comment]
This is looking really good! I made on suggestion that will fix the formatting of the CSV. if you add it, make sure you use git pull to get those changes synced locally **accepted suggestion**

The notebook was hard to comment on inline so adding observations/suggestions/requests here:

Labels will be edited in a figure editor right? if so, the size of text, italics, and periods in the names are fine **yes, they are fixed in figure editor**
will you also make a legend for the side bar colors? **yes, added in figure editor**
can you check whether the red/blue palette is color blind friendly? **checked with Megan, these hex codes should be friendly**
can you put the sessionInfo block at the bottom of the notebook and then restart the kernel and re-run the notebook and push your changes? it looks like you have extra libraries loaded up that you didn't end up using. **moved sessioninfo and updated notebook**

**Review by taylorreiter — APPROVED** [review]
Ok thank you for all of the changes! We can merge this now and update the readme later if you want to, or we can wait and add those changes to this PR later

**Pull request merged**